buildscripts/buildvars-setup.cmd

```diff
@@ -159,11 +169,6 @@ echo See: https://github.com/dotnet/corert/blob/master/Documentation/prerequisit
 exit /b 1
```

Comment:
Nit: Could you please mention 2019 in https://github.com/dotnet/corert/blob/master/Documentation/prerequisites-for-building.md as well?

Reply:
Done. Also fixed up the link for the VS2017 page.